test/removeNPMAbsolutePaths.js

```diff
@@ -149,6 +149,28 @@ describe('removeNPMAbsolutePaths.js', function () {
           });
       });
 
+      it('rewrite pacakge.json, but only specific fields', function () {
+        const dirPath = `${__dirname}/data/specific_fields`;
+        const filePath = `${dirPath}/module/package.json`;
+        const opts = {
+          userSpecifiedKeys: true,
+          keys: ['_inBundle', '_where'],
+        };
+        const promise = removeNPMAbsolutePaths(dirPath, opts);
+        return expect(promise).be.fulfilled
+          .then((results) => {
+            expect(results).to.be.an('array').that.have.lengthOf(3);
+            const fileResults = results.find(result => result.filePath === filePath);
+            expect(fileResults).to.include({ success: true, rewritten: true });
+            expect(fileResults.err)
+              .to.not.exist;
```

Comment:
This shouldn't be in it's own line. It should be `expect(fileResults.err).to.not.exist;`

Reply:
oops sorry must have just fat-fingered it.